plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/plugin.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="org.eclipse.wst.server.core.runtimeModuleType">
+      <runtimeModuleType
+            id="com.google.cloud.tools.eclipse.appengine.standard.runtime.additions.java8"
+            runtimeTypes="com.google.cloud.tools.eclipse.appengine.standard.runtime">
+         <moduleType
+               types="jst.web"
+               versions="3.0,3.1">
+         </moduleType>
+      </runtimeModuleType>
+   </extension>
+
+   <extension
+         point="org.eclipse.wst.common.project.facet.core.groups">
+      <members
+            group="com.google.cloud.tools.eclipse.appengine.facets.standard.constraints.servlet">
```

Comment:
So if I follow you, these two constraints groups are being contributed to by two different plugin.xml files. It's not redefining the group here. Is that right?